source/_docs/installation/synology.markdown

```diff
@@ -175,68 +274,218 @@ case $1 in
+1. Click on "*Create*" > "*Triggered Task*" > "*User-defined script*"
+1. In "*Task*" write "**Home Assistant**"
+1. Click on the checkbox next to "*Enabled*"
+1. Make sure "*root*" is selected in "*User*"
+1. Go to "*Task Settings* settings
+1. In "*User-defined script*" write "**/volume1/homeassistant/hass-daemon start**"
+1. Click on "*OK*"
 
+---
 
-Copy your configuration.yaml file into the config folder
-That's it... you're all set to go
+## {% linkable_title Using older Python 3 provided by Synology %}
 
-Here are some useful commands:
+<p class="note warning">
+Synology only provides [Python 3.5.1](https://www.synology.com/en-global/dsm/packages/py3k), which is not compatible with Home Assistant 0.65.0 or later.<br/>
+The instructions above describes how to install newer Home Assistant on your Synology NAS.<br/>
+If you want to proceed with a older version of Home Assistant, follow the instructions below. Please note that you may not be able to use certain components, such as "[Cloud](/components/cloud/)" and [Homekit](/components/homekit/)".
+</p>
 
-- Start Home Assistant:
+Running these commands will:
+* Install Home Assistant 0.64.3
+* Enable Home Assistant to be launched on http://*Synology_IP*:8123
 
-```bash
-$ sudo /volume1/homeassistant/hass-daemon start
-```
+Using the [Synology webadmin](https://www.synology.com/en-global/knowledgebase/DSM/help):
+* Install Python 3 using the Synology Package Center
+* Create "homeassistant" user
+* Create "homeassistant" [Shared-Folder](https://www.synology.com/en-global/knowledgebase/DSM/help/DSM/AdminCenter/file_share_desc)
+* Enable [SSH](https://www.synology.com/en-global/knowledgebase/DSM/help/DSM/AdminCenter/system_terminal) service
+* Optionally start Home Assistant on bootup of your Synology NAS
+
+### {% linkable_title Install Synology provided Python 3 %}
+
+Install Python 3 as follows:
+1. Open "[*Package Center*](https://www.synology.com/en-global/knowledgebase/DSM/help/DSM/PkgManApp/PackageCenter_desc)"
+1. Go to "*All Packages*"
+1. Scroll all the way down until you find "*Python3*" and click on "*Install*"
+
+Follow the other instructions at "[Using the Synology webadmin](#using-the-synology-webadmin)", ignore the  "Python 3 package" part.
 
-- Stop Home Assistant:
+### {% linkable_title Installing Home Assistant 0.64.3 %}
 
+After the Python 3 package has been installed, open terminal and open SSH to the synology.<br/>
+Replace "*user*" with your Synology NAS user and "x.x.x.x" with the its IP address:
 ```bash
-$ sudo /volume1/homeassistant/hass-daemon stop
+$ ssh user@x.x.x.x
 ```
+Install `pip` (Python’s package management system):
+```bash
+# sudo /volume1/@appstore/py3k/usr/local/bin/python3 -m ensurepip
+```
+Use `pip` to install Homeassistant 0.64.3:
+```bash
+# sudo /volume1/@appstore/py3k/usr/local/bin/python3 -m pip install homeassistant==0.64.3
+```
+Create a file named "**hass-daemon**" in your "*homeassistant*" Shared-Folder with the script below as its content.
```

Comment:
Create a file named `hass-daemon` in your `homeassistant` Shared-Folder with the script below as its content.